dashboard/src/modules/components/SidebarComponent/sideMenuOptions.js

```diff
@@ -21,7 +21,7 @@ export const menuOptions = [
         submenu: true,
         key: "all_runs",
         submenuOf: "dashboard",
-        link: "all_runs",
+        link: "all-runs",
```

Comment:
I suppose this applies to all of the `link` fields: but should we be using the constants you've defined, like `ALL_RUNS`, here so we don't have to worry about link strings that differ from the routes again?